screens/Search/index.tsx

```diff
@@ -125,6 +125,7 @@ const FindFriends = () => {
                     <CustomButton
                       variant="outline"
                       title="Add"
+                      fontWeight="semibold"
```

Comment:
this is not being applied and it is affecting the fontWeight added inside, I suggest you remove it